### DISCUSSION TITLE
プロジェクト管理についての会話スレッド

### DISCUSSION THREAD

**Discussion by Ayato-kosaka** [Ayato-kosaka/react-itinerary#707] [General]
プロジェクト管理についての会話チャンネル

**Comment by Ayato-kosaka** [comment]
@Takapy1 
ディレクトリ構成わかりやすくしてみたから一読しておいて〜
気になる所あればおしえて〜
https://docs.google.com/document/d/1jAuauX2I8qd7qan1qpyIRQV2hGdJ1BN3HJooKbafRSM/edit#

**Reply by Ayato-kosaka** [reply]
@Takapy1 
./4.Marketing/5.Monetize_Design で Admob は管理するから、./1.Project_Management/Finance/Admob 不要だったら消しておいて〜

---
./4.Marketing/Movie も、./6.Finace/Crowdfunding に入れるがいいかな〜

**Comment by Ayato-kosaka** [comment]
@Takapy1 
プロジェクト管理方針についても更新して、マーケティング指向プロダクト開発についてまとめてみたから一読しておいて〜
不明点あれば明日認識合わせしよう！
https://docs.google.com/document/d/18DeDMI2U_MpkIX1FtwyuGc6s_4oMWqIxGrnAB6wa_lg/edit#

**Reply by Takapy1** [reply]
@Ayato-kosaka 
まだみれてない！

**Comment by Ayato-kosaka** [comment]
@Takapy1 
先週の進捗会の反省点で上がってた、来週することざっと起票しておく件、時間があれば対応しておいて〜
今週分のタスクの移動は進捗会でやるから大丈夫！

**Reply by Takapy1** [reply]
@Ayato-kosaka 
了解です。
ただ、ちょっとまだできてない、間に合わんかも！

**Comment by Ayato-kosaka** [comment]
@Takapy1
明日の進捗確認も短時間で済ませたいから、時間があれば、来週することざっと起票しておいて〜
コストとかも打ち合わせで確認するから、事前にざっと降っておいて問題ない〜

**Reply by Takapy1** [reply]
@Ayato-kosaka 
おけですー！

**Comment by Ayato-kosaka** [comment]
@Takapy1 
Sprint Goal と PBI Goal 記載したから目を通しておいて〜

**Reply by Takapy1** [reply]
@Ayato-kosaka 
ありがとう

**Reply by Ayato-kosaka** [reply]
@Takapy1 
ごめん、前回の打ち合わせで、sprint backlog の話忘れてたー
Week 38 に映すものは移しておいてー
今スプリントでたいおうしそうなもの no week に作っておいたー
